setup.py

```diff
@@ -35,7 +35,7 @@
         'speaklater>=1.2',
     ],
     classifiers=[
-        'Development Status :: 4 - Beta',  # TODO: update dev status
+        'Development Status :: 4 - Beta',
```

Comment:
We should add `'Programming Language :: Python :: 3'` here - and no `:: 2` to send the clear message of support.